src/model/QMLFrontend.h

```diff
@@ -183,6 +185,20 @@ QObject *singletonGetterByFunction(QQmlEngine *engine, QJSEngine *scriptEngine)
 }
 
 
+/*!
+ * Register the given interface QML implementation as an ucreatable QML type.
```

Comment:
does not register a QML implementation

Reply:
That's what I always say. It is basically copy-pasted from the other registration functions docu, that you wrote. I always stress that what is actually registered is not the interface implementation but the frontend type. You are a bit vague here...